cves/CVE-2015-1831.yml

```
@@ -144,10 +146,10 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: This commit was interestion because they seemed to know that the excludeParams wasn't working as intended but the way they went about to add addtional exluded patterns was wrong.
   commits:
-    - commit:
-      note:
+    - commit: 89cbe13853a849340d740d45685e6fd14da93d9b
+      note: Added option to define additional accepted/ecluded patterns
```

```suggestion
      note: Added option to define additional accepted/excluded patterns
```
As Gavriel said, it may be a good idea to spellcheck before you commit.

---

I went through and fixed all the spelling.